### ISSUE TITLE
[Research] OUI Compliance audit of the `expressions` plugin

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4081]
Tracking issue for OUI compliance audit of the `expressions` plugin.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

<details>
  <summary>Related files</summary>

  size (bytes) | file | notes
---|---|---
450 | [src/plugins/expressions/public/_expression_renderer.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/expressions/public/_expression_renderer.scss) | 
224 | [src/plugins/expressions/public/index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/expressions/public/index.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.

**Comment by BSFishy** [comment]
Finished. #4122
